lib/generators/spree/retail/install/templates/config/initializers/spree_retail.rb

```diff
@@ -0,0 +1,4 @@
+# Spree Retail Setup
```

Comment:
@DanielWright What do you think of this? 
Everywhere we use `spree` instead of `solidus`.